### PR TITLE
fix: Restore Storybook-based e2e tests

### PR DESCRIPTION
**Pull request by thure**
This PR restores e2e tests that depend on Storybook:
- `react-client`
- `react-shell`

### CODE REVIEW

File: packages/sdk/react-client/src/testing/ClientRepeater.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  className?: string;
+  Component: FC<any>;
+  args?: Omit<P, 'id' | 'count'>;
+};
+
+/**
+ * Utility component for Storybook stories which sets up clients for n peers.
+ * The `Component` property is rendered n times, once for each peer.
+ */
+// TODO(burdon): Reconcile with ClientSpaceDecorator.
```
Comment (wittjosiah) [inline packages/sdk/react-client/src/testing/ClientRepeater.tsx:31]: should also delete `ClientSpaceDecorator`

File: packages/sdk/react-shell/src/playwright/invitations-manager.ts [hunk 2]
```diff
@@ -150,7 +150,7 @@ export class InvitationsManager extends ShellManager {
       await peer.getByTestId('join-identity').click();
     }
     await peer.getByTestId(`${type === 'device' ? 'halo' : 'space'}-invitation-input`).waitFor({ timeout: 500 });
-    await peer.getByTestId(`${type === 'device' ? 'halo' : 'space'}-invitation-input`).type(invitation);
```
Comment (wittjosiah) [inline packages/sdk/react-shell/src/playwright/invitations-manager.ts:153]: may be worth taking a peek at the shell manager as well to see if it's using `type`, perhaps that causes some flakiness there too

File: packages/sdk/react-shell/src/stories/Invitations.e2e-stories.tsx [hunk 3]
```diff
@@ -112,7 +112,9 @@ const Panel = ({ id, panel, setPanel }: { id: number; panel?: PanelType; setPane
   }
 };
 
-const Invitations = ({ id }: { id: number }) => {
+const Invitations = (args: { id: number; count: number }) => {
+  const { id }: { id: number } = args;
+  console.log('[invitations]', args);
```
Comment (wittjosiah) [inline packages/sdk/react-shell/src/stories/Invitations.e2e-stories.tsx:117]: 👀

**Review by wittjosiah — APPROVED** [review]

File: packages/sdk/react-shell/src/playwright/invitations-manager.ts (hunk 2 above)
Comment (thure) [reply packages/sdk/react-shell/src/playwright/invitations-manager.ts:153]: I looked for other uses of `.type` when I found this, but didn't find any.

**Pull request merged**